spark-thrift-server/templates/deployment.yaml

```diff
@@ -35,12 +35,18 @@ spec:
             - "spark.datasource.gitbase.user={{ .Values.app.gitbaseUser }}" 
             - --conf 
             - "spark.datasource.gitbase.password=$(GITBASE_PASSWORD)"
+            - --conf
+            - spark.driver.host=$(POD_IP)
```

Comment:
nit: any reason why this is not quoted as the rest of the confs?

Reply:
Good point